### PR TITLE
Add URLs for pages previously ignored in xref generation

### PR DESCRIPTION
**Pull request by markquintontulloch**

### CODE REVIEW

File: resourceDescriptors.yaml [hunk 1]
```diff
@@ -574,12 +670,33 @@
...
+  pages:
+    - name: gene/interactions
+      url: https://pdbj.org/mine/summary/[%s]
+
 - db_prefix: PMID
   name: PubMed
   aliases: ['pubmed']
   example_gid: PMID:26447131
-  gid_pattern: "^PMID:\\d+$"
+  gid_pattern: "^(PMID|pubmed):\\d+$"
```
Comment (sierra-moxon) [inline resourceDescriptors.yaml:686]: 👍

**Review by sierra-moxon — APPROVED** [review]

**Comment by azurebrd** [comment]
@sierra-moxon @markquintontulloch  If the gid_pattern is generally used in a case-sensitive way, could we also allow 'PubMed' ?  I think that's how I've seen it most of the time when it's not 'PMID'.  If it's generally used case-insensitive, then nevermind :)

**Comment by markquintontulloch** [comment]
@azurebrd I went with the lower case pubmed as that's how it was specified in genetic_interaction_etl.py.  I believe the prefixes are generally all lowercase in the PSI-MITAB file format.

**Comment by azurebrd** [comment]
@markquintontulloch Ah, gotcha.  I didn't realize the prefix casing was standardized in PSI-MITAB files, @chris-grove recently found an issue where the code was expecting the DOI prefix in a different casing than a source had, so we changed the code instead of asking the source provider.  I imagine most places will have PMID instead of 'PubMed' or 'pubmed', but I didn't remember about interaction files having them, so I'm sure there's a lot more out there.  Thanks Mark.

**Pull request merged**